pkg/kubecost/query.go

```diff
@@ -56,6 +56,14 @@ type AllocationQueryOptions struct {
 	Step                    time.Duration
 }
 
+type AccumulateByType string
```

Comment:
Let's talk naming! (This works, and we may well go with it. Just want to explore a couple other options, or hear if you had other options in mind.)